app/src/lib/stores/app-store.ts

```diff
@@ -352,23 +358,10 @@ export class AppStore {
     this.emitQueued = false
     const state = this.getState()
 
-    this.emitter.emit('did-update', state)
+    this._emitter.emit('did-update', state)
```

Comment:
Minor thing, but we should call `super.emitUpdate(state)` here in case the underlying `eventName` is changed.